dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/CommunityMatcher.java

```diff
@@ -31,8 +31,6 @@ private CommunityMatcher() { }
                 hasJsonPath("$.uuid", is(uuid.toString())),
                 hasJsonPath("$.handle", is(handle)),
                 hasJsonPath("$.type", is("community")),
```

Comment:
again better overload the method than reduce the checks

Reply:
I agree here as well.  Instead of removing the checks for embedded content from `matchCommunityEntry()`, we should create a separate method named `matchCommunityEntryNoEmbeds()` (or similar) which does this.

Reply:
@abollini : This was resolved in a different manner.  A new `matchCommunityEntryFullProjection()` method was added, and all existing tests that depend on embeds (full projection) were updated to use that new method.